packages/docs/src/stories/templates/teaser-media.stories.ts

```diff
@@ -10,19 +10,25 @@ export default {
       type: 'figma',
       url: 'https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=3510-218627&t=lJxrBJPRziV74fnu-4'
     }
-  }
+  },
+  decorators: [
+    (story: any) => html`
+      <style>
+        #anchor--templates-teaser-media--teaser-media-overrides .innerZoomElementWrapper {
+          overflow: hidden;
+        }
+      </style>
+      ${story()}
+    `
+  ] as unknown
 };
 
 export const TeaserMediaWithLink = {
   render: () =>
     html` <div class="flex flex-col gap-12">
       <sd-teaser-media variant="gradient-dark">
         <div slot="media" class="relative">
-          <img
-            class="w-full aspect-ratio"
-            src="./placeholders/images/coins.jpg"
-            alt="Close-up of hands stacking coins into small piles on a table, suggesting financial planning or saving."
-          />
+          <img class="w-full aspect-ratio" src="./placeholders/images/coins.jpg" alt="" />
```

Comment:
Not sure, if these really count as decorative. Would love to hear Debbies thoughts on that: https://solid-design-system.github.io/solid/fix_sd-teaser-and-teaser-media-a11y/?path=/docs/templates-teaser--docs

Reply:
As in the examples the images are not decorative but delivering emotions (e.g. Future is an attitude -> man with young children) I would actually add a meaningful alt attribute.
I suggest to add a small a11y hint though which says something like in case your used image is decorative keep it empty...